README.md

```diff
@@ -0,0 +1,29 @@
+# Exclusive Relayer Configurations
+
```

Comment:
NIT: We should add some more generic description on what this repo is and then point to below steps/guide on how to be added to the exclusivity set. But this can also be done later in a subsequent PR